pkg/option/config.go

```diff
@@ -1755,8 +1756,8 @@ type DaemonConfig struct {
 	// CiliumNode resource for the local node
 	AutoCreateCiliumNodeResource bool
 
-	// ipv4NativeRoutingCIDR describes a CIDR in which pod IPs are routable
-	ipv4NativeRoutingCIDR *cidr.CIDR
+	// Ipv4NativeRoutingCIDR describes a CIDR in which pod IPs are routable
+	Ipv4NativeRoutingCIDR *cidr.CIDR
```

Comment:
nit, this could be renamed to `IPv4NativeRoutingCIDR` (instead of `Ipv4NativeRoutingCIDR`)

Reply:
Yes, I used that variable name purposefully because there was a function with the same name which was being called with `DaemonConfig` struct.